autonomx/layout/LatticeView.qml

```diff
@@ -27,6 +27,7 @@ Item {
     // backend read
     onGeneratorIndexChanged: {
         panManager.setFromLatticeView(generatorModel.at(generatorIndex));
+        allowSnapW = allowSnapH = false
```

Comment:
I would put that statement in two separate lines and add a semicolon after each.

allowSnapW = false;
allowSnapH = false;